plugin/inlay_hint.py

```diff
@@ -101,9 +101,9 @@ def get_inlay_hint_html(view: sublime.View, inlay_hint: InlayHint, session: Sess
 
 def format_inlay_hint_tooltip(tooltip: Optional[Union[str, MarkupContent]]) -> str:
     if isinstance(tooltip, str):
-        return tooltip
+        return html.escape(tooltip)
     if isinstance(tooltip, dict):  # MarkupContent
-        return tooltip.get('value') or ""
+        return html.escape(tooltip.get('value') or "")
```

Comment:
I would probably move this escape to the location where this content is inserted into HTML as now this code assumes that it will be used in HTML.

Reply:
Well, never mind. I see that this way it's more consistent with its sister function `format_inlay_hint_label`.
If I'd want to be picky I could probably rename those to something like `get_inlay_hint_label_html` and `get_inlay_hint_tooltip_html` but I'll let it go already. :)